### DISCUSSION TITLE
Scaling heading font sizes

### DISCUSSION THREAD

**Discussion by Ian-Hitt** [Audentio/components#85] [General]
I'm not a big fan of scaling font size headings, for a few reasons.

1. Hard to use headings consistently since they change depending on what VW I'm currently developing at.
2. No other font sizes scale. So headings get huge, but the accompanying text stays the same
3. we use rems for everything (instead of ems) so headings get bigger, but their paddings/margins/etc.. don't scale with them
4. My H5s at full width of my monitor are like 26px and look pretty awful
5. We wouldn't want to scale _all_ typography (like, canvases/menus should be left alone), but then they would be inconsistent w/ the rest of the UI.

Proposal: Just stop scaling headings based on viewport width. Set them to the best legible size that has consistency w/ the rest of the UI.

**Comment by mikecreuzer** [comment]
Font size should change based on viewport width. Headings and text. So I dont agree with completely ditching this. But we likely need to do a bit of digging into the ideal way text should scale. 

A site like this, modern example, indeed uses typography scaling. https://www.galacticenergy.com/ (was just skimming awwwards)

Any good framework should be typography first. So this would likely be a sweeping change whenever we implement a change. 

My question for now is what is the issue? Are we worse off? I mean is there something Stuff does that gets in the way of how youd normally scale fonts if you needed something at a specific size?

**Comment by mikecreuzer** [comment]
This is an interesting read. https://betterwebtype.com/articles/2019/05/14/the-state-of-fluid-web-typography/

I think the gated approach without fluidity is likely what we should use (and I believe currently do)
